Bus.h

```diff
@@ -0,0 +1,38 @@
+//
+// Created by Anoush Movsisian on 26.03.24.
+//
+
+#ifndef BUSES_1_BUS_H
+#define BUSES_1_BUS_H
+
+#include "Crossroad.h"
+
+class Bus {
+private:
+    Crossroad source;
+    Crossroad dest;
+    int duration;
+
+public:
+    Bus() = default;
+    Bus(Crossroad _s, Crossroad _d) : source(source), dest(_d), duration(0) {}
```

Comment:
maybe you should try giving _s to source? source(_s)

Reply:
Yes thank you, will fix it right away